src/he2hb.cc

```diff
@@ -135,7 +134,7 @@ void he2hb(
 
             lapack::Queue* queue = A.compute_queue( panel_device, queue_0 );
 
-            int64_t nb       = A.tileNb(0);
+            int64_t nb       = func::max_blocksize(A.nt(), A.tileNbFunc());
```

Comment:
This NB is just for determining the workspace size.  But yea, basically everything that involves banded matrices is going to need some reworking.  (I suspect `he2hb` might currently work if `tileMb(i) <= tileMb(i+1)` for all `i`.  But, that's not particularly useful without `hb2st`.)

There's a related can of worms involving `geqrf` and `getrf_tntpiv` since they assume that a process's top tile is either it's only tile or `mb >= nb`.